src/osemgrep/targeting/Glob_lexer.mli

```diff
@@ -0,0 +1,16 @@
+(*
+   Parse a glob pattern according to glob(3) and glob(7), which is the
+   POSIX standard for old-fashioned shell globbing patterns for matching
+   file paths. This is what gitignore uses.
+
+   Examples:
+
+   *.c          # all local files with a '.c' extension
+   /tmp/**      # all valid paths under '/tmp/'
+   Thing.ml?    # matches 'Thing.ml' as well as 'Thing.mli', 'Thing.mll', etc.
+   [a-z0-9]     # matches a single character in these ranges
```

Comment:
** was also part of the old-fashioned shell globbing syntax?

Reply:
No, I don't think it is. It's documented by gitignore but I can't find a mention of this in the man pages for glob or fnmatch. I fixed the comment above.